pkgs/applications/graphics/xpano/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, stdenv
+, fetchurl
+, cmake
+, pkg-config
+, ninja
+, opencv
+, SDL2
+, gtk3
+, catch2_3
+, spdlog
+, exiv2
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xpano";
+  version = "0.16.1";
+
+  src = fetchurl {
+    url = "https://github.com/krupkat/${pname}/releases/download/v${version}/${pname}-v${version}.tar.gz";
+    sha256 = "sha256-2oh5FGP192qp/0H6G0d6kpyfukonbSQq1BSzWoR1ghk=";
+  };
+
+  patches = [
+    # force install desktop + icon files
+    ./skip_prefix_check.patch
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    opencv
+    SDL2
+    gtk3
+    spdlog
+  ] ++
+  lib.optionals (lib.versionAtLeast exiv2.version "0.28.0") [ exiv2 ];
+
+  checkInputs = [
+    catch2_3
+  ];
+
+  doCheck = true;
+
+  cmakeFlags = [
+    "-DBUILD_TESTING:BOOL=ON"
```

```suggestion
    "-DBUILD_TESTING=ON"
```
that's just a type information for cmake gui

---

Done